### PR TITLE
fix: remove unsued dependencies

### PR DESCRIPTION
**Pull request by clarkdo**
@ScriptedAlchemy Looks like all the dependencies are not used in this package anymore, this pr is to remove them for eliminate unnecessary nested and hoisted node_modules

### CODE REVIEW

**Comment by clarkdo** [comment]
@ScriptedAlchemy Sorry, I missed that part, I've updated pr

File: yarn.lock [hunk 1]
```diff
@@ -1930,39 +1914,14 @@ babel-preset-jest@^24.9.0:
     "@babel/plugin-syntax-object-rest-spread" "^7.0.0"
     babel-plugin-jest-hoist "^24.9.0"
 
-babel-runtime@^6.22.0, babel-runtime@^6.23.0, babel-runtime@^6.26.0:
+babel-runtime@^6.23.0:
   version "6.26.0"
   resolved "https://registry.yarnpkg.com/babel-runtime/-/babel-runtime-6.26.0.tgz#965c7058668e82b55d7bfe04ff2337bc8b5647fe"
   integrity sha1-llxwWGaOgrVde/4E/yM3vItWR/4=
   dependencies:
     core-js "^2.4.0"
```
Comment (midnight-wonderer) [inline yarn.lock]: Should you update `yarn.lock` as well?  
AFAIK, `core-js` should not be a dependency anymore.  
Not sure how `yarn.lock` works, though.

Refer to https://github.com/ScriptedAlchemy/webpack-external-import/issues/47 for details

Comment (midnight-wonderer) [reply yarn.lock]: Probably used by some devDependencies. NVM if that is the case.

Comment (clarkdo) [reply yarn.lock]: You're right, they are from devDeps

**Comment by ScriptedAlchemy** [comment]
will you run `yarn demo` locally on the repo and browse to localhost:3001, just to verify builds work

**Comment by clarkdo** [comment]
@ScriptedAlchemy Below is screenshot, is this expected ?
Screenshot 2019-11-26 at 22 46 43

**Comment by ScriptedAlchemy** [comment]
Yep, have updated In a beta version on npm

**Pull request closed without merging**